src/shared/components/ncTable/partials/rowTypePartials/RowFormWrapper.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<div class="row space-T">
+		<div class="fix-col-4">
+			<div class="title">
+				{{ title }}{{ mandatory ? '*': '' }}
+			</div>
+		</div>
+		<p v-if="description" class="fix-col-4 span">
+			{{ description }}
+		</p>
+		<div :class="[ `fix-col-${width}` ]" class="slot">
+			<slot />
+		</div>
+	</div>
+</template>
+
+<script>
+
+export default {
+	name: 'RowFormWrapper',
+	props: {
+		mandatory: {
+			type: Boolean,
+			default: false,
+		},
+		title: {
+			type: String,
+			default: '',
+		},
+		length: {
+		      type: Number,
+		      default: null,
+		    },
+		maxLength: {
+		      type: Number,
+		      default: null,
+		    },
+		description: {
+		      type: String,
+		      default: null,
+		    },
+		width: {
+		      type: Number,
+		      default: 4,
+		    },
```

Comment:
Indentation seems off, maybe we need to check the eslint configuration as this should be auto-fixable?